elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/SQLQueryEngine.java

```diff
@@ -297,7 +303,8 @@ public String getTableVersion(Table table, Transaction transaction) {
 
         Pagination pagination = query.getPagination();
         if (returnPageTotals(pagination)) {
-            queries.add(toPageTotalSQL(expandedQuery, sql, dialect).toString());
+            SQLQuery paginationSql = toPageTotalSQL(expandedQuery, sql, dialect);
+            queries.add(paginationSql == null ? "" : paginationSql.toString());
```

Comment:
We don't want to add a query here if we are not running one.  If we return "1" as the rowcount and are not running a query, we'll want to skip adding to the list of queries.